xt/66-cucumber/01-basic/preferences.feature

```diff
@@ -5,7 +5,7 @@
 Feature: Check correct operation of Preferences screen
 
 Background:
-  Given a standard test company
+  Given a fresh standard test company
```

Comment:
This means that each iteration in the examples creates its own test database. Is that a good idea? Or should we make sure the settings get reset at the end of each iteration to prevent the need to create new databases?